### PR TITLE
More Carty Party Vend Qty

### PR DESCRIPTION
**Pull request by MarkNais**
[QoL]
## About the PR 
Adds more stock to the non-hidden Carty Party items. Generally doubled each line item.

## Why's this needed? 
An increase in the count of high-power flashlight modules was requested, and given the recent-ish darkness-buff, seems reasonable.
The rest I increased, because 2 cartridges-each seemed thematically unreasonably low for a vending machine.

I'd happily accept an argument that some numbers (like PDA stock count) should stay low to encourage restock-orders through QM.

## Changelog

```
(u)MarkNstein:
(+)Adjusted CartyParty stock.
```

### CODE REVIEW

**Comment by Sovexe** [comment]
Unsure how I feel about adding even more network diagnostics carts

File: code/modules/vending/vending.dm [hunk 1]
```diff
@@ -1492,19 +1492,19 @@
...
+		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/atmos, 5, cost=40)
+		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/mechanic, 5, cost=40)
+		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/quartermaster, 5, cost=50)
+		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/medical, 5, cost=40)
+		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/genetics, 5, cost=40)
+		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/toxins, 5, cost=50)
+		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/botanist, 5, cost=40)
+		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/janitor, 5, cost=40)
+		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/engineer, 5, cost=70)
+		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/diagnostics, 5, cost=70)
```
Comment (MarkNais) [inline code/modules/vending/vending.dm:1505]:
```suggestion
		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/diagnostics, 2, cost=70)
```
manually revert line

**Comment by MarkNais** [comment]
Back to 2

**Review by flrsh — APPROVED** [review]

**Review by ZeWaka — APPROVED** [review]

File: code/modules/vending/vending.dm [hunk 2]
```diff
@@ -1492,19 +1492,19 @@
...
-		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/quartermaster, 2, cost=50)
-		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/medical, 2, cost=40)
-		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/genetics, 2, cost=40)
-		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/toxins, 2, cost=50)
-		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/botanist, 2, cost=40)
-		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/janitor, 2, cost=40)
-		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/engineer, 2, cost=70)
+		product_list += new/datum/data/vending_product(/obj/item/device/pda2, 20, cost=100)
+		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/atmos, 5, cost=40)
+		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/mechanic, 5, cost=40)
```
Comment (UrsulaMejor) [inline code/modules/vending/vending.dm:1497]: The analysis made easy cartridge is just a cheaper and slightly better version of the networking cartridge. I recommend dumping it from this vendor. If you do, I'll merge this.

**Review by UrsulaMejor — CHANGES_REQUESTED** [review]

**Pull request merged**